content(Fx142): & inside at-scope blocks has no implicit specificity (#40983)
mdn/content: files/en-us/web/css/@scope/index.md

Author: Q: Is this appropriate here:

```yaml
browser-compat:
  - css.at-rules.scope
  - css.selectors.nesting.at-scope
```

Reviewer: I think we should add the css.selectors.nesting.at-scope data to css..at-rules.scope, but, yes, definitely add especially if the top `&` part doesn't show, but would be better to have one table instead of two.

### Specificity in `@scope`

> Inside an `@scope` rule, both bare selectors and [&](/en-US/docs/Web/CSS/Nesting_selector) behave as if `:where(:scope)` were prepended to the selector.

Reviewer: ```suggestion
Inside an `@scope` rule, both bare selectors and the [`&`](/en-US/docs/Web/CSS/Nesting_selector) nesting selector behave as if `:where(:scope)` were prepended to the selector.
```

Reviewer: definitely the back ticks. the rest of the edit is optional

> Because {{cssxref(":where", ":where()")}} has zero specificity, bare selectors and `&` add zero weight and only the specificity of the rest of the selector counts.

Reviewer: ```suggestion
Because {{cssxref(":where", ":where()")}} has zero [specificity](/en-US/docs/Web/CSS/CSS_cascade/Specificity), bare selectors and `&` add zero weight. The specificity weight is determined by the rest of the selector.
```

> An `& img` selector is the equivalent to writing `:where(:scope) img` (0-0-1).

Reviewer: ```suggestion
For example, the specificity of the `& img` selector is equivalent to the specificity of `:where(:scope) img` (0-0-1).
```

> [!WARNING]
> The specificity of `&` inside `@scope` blocks is handled differently according to the browser engine and release version.

> > Check [Browser compatibility](#browser_compatibility) for details.

Reviewer: if you include the table here, we can link to that table instead of to a different page.

> In both cases in the following example, the only specificity comes from `img`:

Reviewer: ```suggestion
In both cases in the following code block, the only specificity comes from `img`:
```

optional, but as there are examples below, i try to avoid using 'example' for stuff not under that heading (and line 194)

}
```

By contrast, using `:scope` explicitly selects the scope root and adds class-level specificity (0-1-0), since `:scope` is a pseudo-class.

> In the following example, `:scope img` has a specificity of 0-1-1:

Reviewer: link pseudo-class (it's not linked before this in the page. you could link the previous one, but this is a better place in terms of content).
optionally change "example" to "code block"
